### DISCUSSION TITLE
[FF7] how to reduce speed in battle

### DISCUSSION THREAD

**Discussion by beb1senate** [julianxhokaxhiu/FFNx#385] [General]
Hello, in 1.10.1.0, speed game too fast. Even when disable or reduce the speed in the game or in game driver. thanks

**Comment by julianxhokaxhiu** [comment]
> Hello, in 1.10.1.0, speed game too fast. Even when disable or reduce the speed in the game or in game driver. thanks

You need to use the latest canary. Please stop creating threads.
